### PR TITLE
feat(cmd): add .taf command

### PR DESCRIPTION
**Pull request by Lauvaan**
## Description

fixes: #434 
Provides the taf weather report for the specified airport.

## Test Results

image

## Discord Username

Lauvaan#3330

### CODE REVIEW

File: src/commands/utils/taf.ts [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+                description: 'You must provide an airport ICAO code.',
+                color: Colors.Red,
+            });
+            await msg.reply({ embeds: [noQueryEmbed] });
+            return Promise.resolve();
+        }
+        const icaoArg = splitUp[1];
+
+        try {
+            const tafReport = await fetch(`https://avwx.rest/api/taf/${icaoArg}`, {
```
Comment (tracernz) [inline src/commands/utils/taf.ts:27]: Should this not use the FBW API that provides caching etc?

Comment (Lauvaan) [reply src/commands/utils/taf.ts:27]: I'm not sure, just used the same one that was used in the .metar command

Comment (benw202) [reply src/commands/utils/taf.ts:27]: We could change this but I see no real need to. Vatsim uses or have used AVWX and we've been very kindly given a plan by AVWX for free.

Am open to changing this if we feel the need - I'm sure there was a discussion about this when we originally chose to go with AVWX, not sure though!

File: .github/command-docs.md [hunk 2]
```diff
@@ -137,6 +137,7 @@
 | .roleinfo    | Lists the guild's current amount of members                                          | ---                                                            |
 | .simbiefdata | Lists a number of attributes from the last generated SimBrief flightplan of a user   | ---                                                            |
 | .station     | Provides station information                                                         | ---                                                            |
+| .taf         | Provides the TAF report of the requested airport                                     | ---
```
Comment (benw202) [inline .github/command-docs.md:140]: Missing the `|`

```suggestion
| .taf         | Provides the TAF report of the requested airport                                     | ---                                                            |
```

**Review by benw202 — COMMENTED** [review]
One small thing, otherwise it's looking great!

**Pull request merged**